CHANGELOG.md

```diff
@@ -15,6 +15,13 @@
 ### General fixes
 
  * Fix accesses to freed metadata cache mutexes on client termination (#3279)
```

Comment:
side note: in .net i'm up to 1.6.2 already, maybe you want to skip a version here, maybe not. i'm not sure which is better / worse.

Reply:
Right, this will most likely be v1.7.0 anyway.